src/modules/user/adapter/db/user-fixture.ts

```diff
@@ -14,13 +14,16 @@ export class UserFixture {
     return this.userRepository.save(user);
   }
 
-  public mockUserEntity(overrides: Partial<User> = {}): Partial<User> {
+  public mockUserEntity(overrides: Partial<User> = {}): User {
     return {
-      discordAvatar: "avatar",
+      id: 0,
+      discordAvatar: "https://picsum.photos/200",
       discordId: "1",
       discordName: "name",
       shortId: this.createShortId(),
       uuid: this.createUUID(),
+      createdAt: new Date(),
+      updatedAt: new Date(),
```

Comment:
I feel that the id, createdAt and updatedAt shouldn't be given here by default, but let the database to populate them. If custom values need to be provided for these fields, then the `overrides` function parameter should be used

Reply:
I initially added them due to some type incompatibilities when mocking a non-existing user entity. Because this user entity is not added to the db, I couldn't cast it to the type `User`. But I guess it doesn't have to be. I changed it [here](https://github.com/across-protocol/scraper-api/pull/83/commits/682334bb81ff20424aefa8494ec88c8bf8806f16).